### PR TITLE
Add devfile support in UD

### PR DESCRIPTION
**Pull request by ashumilova**
Signed-off-by: Anna Shumilova <ashumilo@redhat.com>

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/13067

### CODE REVIEW

**Comment by sleshchenko** [comment]
The following changes should be applied after merging of this PR
https://github.com/eclipse/che/compare/master...sleshchenko:dropWsConfigStub?expand=1

**Comment by ashumilova** [comment]
ci-build

**Comment by ashumilova** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by ashumilova** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by ashumilova** [comment]
ci-test

**Comment by dmytro-ndp** [comment]
ci-test

**Comment by dmytro-ndp** [comment]
ci-test

**Comment by dmytro-ndp** [comment]
ci-test

**Comment by ashumilova** [comment]
Rebasing to master

**Review by akurinnoy — APPROVED** [review]

**Review by olexii4 — APPROVED** [review]

File: dashboard/src/app/workspaces/create-workspace/create-workspace.service.ts [hunk 1]
```diff
@@ -196,6 +189,39 @@ export class CreateWorkspaceSvc {
     });
   }
 
+  /*createWorkspaceFromDevfile(workspaceDevfile: che.IWorkspaceDevfile, attributes: any): ng.IPromise<che.IWorkspace> {
+    const namespaceId = this.namespaceSelectorSvc.getNamespaceId(),
```
[the remark this answers is not in the document]
Comment (ashumilova) [reply dashboard/src/app/workspaces/create-workspace/create-workspace.service.ts:193]: yes, it's for further workspace creation from devfile

**Comment by ashumilova** [comment]
ci-test

**Comment by ashumilova** [comment]
Pushed re-based and squashed commits.
New test failures are the cause of the PR https://github.com/eclipse/che/pull/13361/files.

**Pull request merged**